scripts/repos/suse.template.repos

```diff
@@ -67,16 +67,15 @@ https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifac
 #https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Products/SLE-Product-WE/${releasever_major}-SP${releasever_minor}/${basearch}/product_debug?auth=basic                   SUSE-SLE-Product-WE-Debuginfo-Pool                      -g -p 89
 #https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Product-WE/${releasever_major}-SP${releasever_minor}/${basearch}/update?auth=basic                           SUSE-SLE-Product-WE-Updates                             -g -p 89
 #https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Product-WE/${releasever_major}-SP${releasever_minor}/${basearch}/update_debug?auth=basic                     SUSE-SLE-Product-WE-Debuginfo-Updates                   -g -p 89
-
-https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Backports/SLE-${releasever_major}-SP${releasever_minor}_${basearch}/standard?auth=basic                                  SUSE-Backports-SLE                                      -g -p 89
+#TEMP
+https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Backports/SLE-15-SP4_${basearch}/standard?auth=basic                                                                      SUSE-Backports-SLE                                      -g -p 89
```

Comment:
If the SP5 repo doesn't exist, then [this same comment](https://github.com/Cray-HPE/metal-provision/pull/325/files#r1280070968) goes.. otherwise just resolve/close this comment. :)